### ISSUE TITLE
Support for "tvOS" framework, pod

### ISSUE THREAD

**Issue by fxwx23** [rs/SDWebImage#1368]
Couldn't use SDWebImage with pods for tvOS target.
```
platform :tvos, '9.0'
use_frameworks!

pod 'Alamofire'
pod 'SwiftyJSON'
pod 'SDWebImage'
```

Getting this error.
```
[!] The platform of the target `Pods` (tvOS 9.0) is not compatible with `SDWebImage (3.7.3)`, which does not support `tvos`.
```
What I miss something?
Thank you beforehand.

**Comment by bpoplauschi** [comment]
We haven't release a version with tvOS support yet. If you really need it, you can point to our latest master: `pod 'SDWebImage', :head`

**Comment by fxwx23** [comment]
All right. I'll try to use `:head`.
Thank you for your response.

**Comment by mvpaudrey** [comment]
I added the line `pod 'SDWebImage', :head` in the Podfile but i got this message:

`[!] The platform of the target ``Pods`` (tvOS 9.0) is not compatible with ``SDWebImage (HEAD based on 3.7.3)``, which does not support ``tvos``.`

Did i miss something too ?
Thank you.

**Comment by fxwx23** [comment]
@mvpaudrey Same error still occurred. Did you find any solutions for this?

**Comment by mvpaudrey** [comment]
@fxwx23 no didn't find any other solution

**Comment by jkrzemie** [comment]
Not released yet so you need to
 `pod 'SDWebImage', :git => 'https://github.com/rs/SDWebImage.git'`

**Comment by fxwx23** [comment]
@jkrzemie Thank you so much.

**Comment by mvpaudrey** [comment]
@jkrzemie thanks a lot

**Comment by bpoplauschi** [comment]
Fixed by #1327  - will make a new release soon (3.7.4).
